### DISCUSSION TITLE
[A-L] (2023/24) Foglio 14 - Esercizio 3

### DISCUSSION THREAD

**Discussion by Elia-Belli** [sapienzastudentsnetwork/algebra#284] [Esercizi - Viaggi/Piazza] [answered]
![immagine](https://github.com/sapienzastudentsnetwork/algebra2324/assets/107944476/8dec8efe-033d-4db5-ba5b-8c642212bd67)

**Comment by Elia-Belli** [comment, marked as answer]
# Soluzione Ufficiale
![immagine](https://github.com/sapienzastudentsnetwork/algebra2324/assets/107944476/b6ad38ee-f505-4b1a-99fe-832dc987d7b8)
![immagine](https://github.com/sapienzastudentsnetwork/algebra2324/assets/107944476/81f0802f-9f8f-47af-b95b-23312c0e7a36)

**Comment by CiottoloMaggico** [comment]
Stesse soluzioni che si ottengono mettendo insieme esercizi precedenti #116 #117 
![image](https://github.com/sapienzastudentsnetwork/algebra2324/assets/79638284/5875c4d7-24d4-4f14-b0e9-086ae13b9b33)

**Comment by 18xChris** [comment]
> Stesse soluzioni che si ottengono mettendo insieme esercizi precedenti #116 #117 ![image](https://private-user-images.githubusercontent.com/79638284/288776699-5875c4d7-24d4-4f14-b0e9-086ae13b9b33.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTEiLCJleHAiOjE3MDIxMzY3MTcsIm5iZiI6MTcwMjEzNjQxNywicGF0aCI6Ii83OTYzODI4NC8yODg3NzY2OTktNTg3NWM0ZDctMjRkNC00ZjE0LWIwZTktMDg2YWUxM2I5YjMzLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFJV05KWUFYNENTVkVINTNBJTJGMjAyMzEyMDklMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjMxMjA5VDE1NDAxN1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWEyNjg1NTFjZTdiMDdhMDliNDVjNTYzZjEyYzA0YjEyODc1MTYwN2U0ZWFkNjhkZmViZTRkYTJiMTZiMjkwMjgmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.Bx9Iwg6EM_Lup7qxWjsa0lIVqIEPraawnG9GPtBjl8Y)

non capisco come fa ad uscirti quella soluzione particolare, a me esce (-1,0,1,0,1,0)

**Comment by CiottoloMaggico** [comment]
Per la soluzione particolare ho fissato x2 = 1, x4 = 0, x6 = 0 e mi sono calcolato x1, x2, x3 con le equazioni del sistema
